app/models/msa.js

```diff
@@ -240,7 +240,8 @@ Msa.methods.aminoAcidTranslation = function(cb, options) {
 
 Msa.methods.dataReader = function(file, datatype, cb) {
   // Skip the datareader batch file for fastq and non-coding DNA/RNA files.
-  if (file.indexOf("fastq") != -1 || datatype == 1) {
+  console.log("datatype: ", datatype);
```

Comment:
Using winston here would give us better control over what is printed to logs.

Reply:
This was just a stray console.log from debugging 😞